sktm/patchwork.py

```diff
@@ -65,7 +69,7 @@ def get_mbox_url(self):
         Returns:
             URL pointing at the object's mbox.
         """
-        return self.url + "/mbox"
+        return self.url + self.mbox_sfx
```

Comment:
I'd really like to see this use path manipulation (and also `mbox` instead of `/mbox` as the default argument) instead of dumb concatenation (I have mentioned it in some commits for sure, it handles slashes *and* their absence) but I'll open an issue describing the problem since I'm sure this is not the only example, and we need to get the change in ASAP.

Reply:
Thank you, @veruu. Yes, I remember you mentioning that, but I wasn't sure I understand all the reasons and the way to use it. At least basic concatenation is straight-forward and easy to understand. Let's discuss and agree on the best way in an issue. Merging!